sdk/js/src/Client.ts

```diff
@@ -132,6 +142,11 @@ export class DVCClient implements Client {
                     )
                 ).then((config) => {
                     this.config = config as BucketedUserConfig
+
+                    if (this.options?.enableCloudData || this.config?.project.settings?.cloudEntityData?.enabled) {
+                        saveEntity(this.user, this.environmentKey).then((res) => console.log(`Saved response entity! ${res}`))
```

Comment:
how come we're saving the entity twice in the identify call here?

Reply:
meant to replace the other one